config/config.debug.toml

```diff
@@ -25,6 +25,7 @@ SequencerNodeURI = ""
 [Synchronizer]
 SyncInterval = "5s"
 SyncChunkSize = 100
+TrustedSequencerURI = ""
```

Comment:
this param is not required because the trusted sequencer url must be read from the smc. I suggest to include a method in the etherman called ethMan.GetTrustedSequencerURL().
Here in the config file we can use a boolean to allow sync from the ts.

Reply:
This config is not the `TrustedSequencerURL` which is supposed to be the `URL` of the `JSON RPC Server`. 
This is the `IP:PORT` of the `Broadcast Service` provided by the `Trusted Sequencer`

I agree to get this info from the `SMC`, but looks like the current information provided by the `SMC` is not enough to fulfill this feature needs.

An option would be to create a new custom `JSON RPC Server` endpoint like `hez_getBroadcastURI`, so we can query this from the `Trusted Sequencer` after getting its `URL` from the `SMC`.

wdyt @arnaubennassar?

Reply:
We end up going with the solution to create a new endpoint to provide the broadcast URI.